CONTRIBUTING.md

```diff
@@ -68,7 +68,7 @@ for more detail on available tox commands.
 
 ### Benchmarks
 
-Performance progression of benchmarks for packages distributed by OpenTelemetry Python can be viewed as a [graph of throughput vs commit history](https://opentelemetry-python.readthedocs.io/en/latest/benchmarks/index.html). From this page, you can download a JSON file with the performance results.
+Performance progression of benchmarks for packages distributed by OpenTelemetry Python can be viewed as a [graph of throughput vs commit history](https://opentelemetry-python.readthedocs.io/en/latest/performance/benchmarks.html). From the linked page, you can download a JSON file with the performance results.
```

Comment:
Why not just stick the actual link here?

Reply:
No particular reason, more so for maintainability of the link since I'm assuming we are more likely to stick with readTheDocs than gh-pages going forward? But I can change it to the other link if we want!